src/classes/pilot/Pilot.ts

```diff
@@ -413,9 +414,10 @@ class Pilot implements ICloudSyncable {
 
   public SetOwnedResource(userCognitoId: string): void {
     console.log('pilot call, set owned resource')
-    this.CloudID = this.ID
-    this.CloudOwnerID = userCognitoId
+    Vue.set(this, 'CloudID', this.ID)
+    Vue.set(this, 'CloudOwnerID', userCognitoId)
```

Comment:
Use `Vue.set` so the Vault Dialog detects the mutation, computing the new Vault Code.